os-sso72/added/openshift-launch.sh

```diff
@@ -11,11 +11,28 @@ function clean_shutdown() {
   wait $!
 }
 
+# CLOUD-2453 Connect-retry loop to wait for the service to become reachable over network
+function wait_for_service() {
+  local -r CURL_CONN_REFUSED_EXIT_CODE="7"
+  if [ -n "${SERVICE_WAIT_HOST}" -a -n "${SERVICE_WAIT_PORT}" -a -n "${SERVICE_WAIT_MESSAGE}" -a -n "${SERVICE_WAIT_RETRY_PERIOD}" ]; then
```

Comment:
Would it make sense to default to some value for '${SERVICE_WAIT_RETRY_PERIOD}'? In case that we want document this feature and make it available also for non-template users... In that case I would also add unit that should be used with this env. Eg: SERVICE_WAIT_RETRY_PERIOD_MILLIS.